dotCMS/src/main/java/com/dotcms/http/CircuitBreakerUrlBuilder.java

```diff
@@ -0,0 +1,104 @@
+package com.dotcms.http;
+
+import java.util.Map;
+
+import org.apache.http.client.methods.HttpDelete;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.client.methods.HttpPost;
+import org.apache.http.client.methods.HttpPut;
+import org.apache.http.client.methods.HttpRequestBase;
+
+import com.dotcms.http.CircuitBreakerUrl.Method;
+import com.dotmarketing.business.DotStateException;
+import com.dotmarketing.util.Config;
+import com.google.common.collect.Maps;
+
+import net.jodah.failsafe.CircuitBreaker;
+
+
```

Comment:
It'd be great to have some Javadoc in these classes and methods.

Reply:
there is a little bit :
https://github.com/dotCMS/core/pull/15744/files#diff-f9f01b5a51d820ce2b06e16e3105eb10R34